### DISCUSSION TITLE
Remove/hide timers

### DISCUSSION THREAD

**Discussion by petyakh** [EdgeTX/edgetx#1546] [Ideas]
I've never met a person who uses the timers, but if you take a look into the model settings it seems to be a very important thing though. Because they take up a lot of space.  So why don't we make the option to hide timers from the model setting or remove them at all?

**Comment by pfeerick** [comment]
And I've seen twice as many who do :-P But you have a valid point, and it is something we want to improve generally... i.e. this is one concept design ...

![image](https://user-images.githubusercontent.com/5500713/152258712-41b27f4a-200f-4994-ac9d-d82691c0ebea.png)

... Now I'm not saying is it will look look, but it does fit with what we had in mind already, which was grouping of settings into their own subpages so the pages aren't as long.
